poster/src/post_with_retries.ts

```diff
@@ -36,6 +36,10 @@ async function postWithRetries(transaction: TransactionConfig, signerKey: string
 
   console.log(`Posting from account: ${pubKey.address}`);
 
+  // TODO: Why manual nonce management?
```

Comment:
Specifically because we want to be amenable to high gas prices and mempool shedding